test/unity/toolkits/drawing_classifier/test_dc_prediction.cxx

```diff
@@ -99,7 +99,8 @@ gl_sframe set_up_perform_inference(
     test_drawing_classifier& mock_model,
     std::unique_ptr<mock_model_backend>& mock_backend,
     std::unique_ptr<mock_compute_context>& mock_context, size_t batch_size,
-    const size_t num_of_rows, const size_t num_of_classes) {
+    const size_t num_of_rows, const size_t num_of_classes,
+    const std::string feature_column_name) {
```

Comment:
I don't know why you need this variable regarding the fact that you already have `feature_list` with size one. If you need this variable, why you change the original single feature variable to a `feature_list`?

Reply:
Okay sure, we can just use the feature column name from the features list and not add another state variable. 

We never changed the original single feature variable to a feature list. We started with a feature list (see https://github.com/apple/turicreate/pull/2439#discussion_r334133796). I agree that at some point, we should change this to be just a `feature_column_name`, but that would require a sizable code change on the `init_model` and `export_to_coreml` side of things which is not related to this PR. I believe that should come as a separate PR altogether.

Reply:
Be brief. Then stick to `feature_list`. Adding a new var referring to the first element of the list adds more complexity for reading the code. I'd better ensure the state is tight and simple.

Reply:
^Yes, I agree.